__tests__/index.spec.ts

```diff
@@ -41,6 +41,13 @@ describe('mande', () => {
     expect(fetchMock).toHaveFetched('/api/2')
   })
 
+  it('can use get without parameters', async () => {
```

Comment:
Can you also add a test that you can pass options to `api.get({ ... })`?

Reply:
Sure thing, I’ll add it.